src/value.zig

```diff
@@ -1711,6 +1729,18 @@ pub const Value = extern union {
                     return Tag.aggregate.create(arena, field_vals);
                 },
             },
+            .Union => switch (ty.containerLayout()) {
+                .Auto => unreachable, // Sema is supposed to have emitted a compile error already
+                .Extern => unreachable, // Handled by non-packed readFromMemory
+                .Packed => {
+                    const union_field = ty.unionFields().values()[0];
```

Comment:
My reasoning here was that when we're trying to `@bitCast` a value such as an integer to a `packed union` then basically we're just putting it in there starting at the top obviously and that's why I'm just picking the first field.